### PR TITLE
Metres of the Lakkhana-suttanta

### PR DESCRIPTION
**Pull request by JJM317**
Another toughie for me as the PDF is a complication of five different articles. But, they are found in a collected essays book, maybe? I pu tthis under articles, maybe it should be excerpts?

### CODE REVIEW

File: _content/articles/metres-of-the-lakkhana-suttanta_norman.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: "The Metres of the Lakkhana-suttanta DN30"
```
Comment (buddhist-uni) [inline _content/articles/metres-of-the-lakkhana-suttanta_norman.md:2]: Proper diacritics please

File: _content/articles/metres-of-the-lakkhana-suttanta_norman.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+course: canonical-poetry
+tags:
+  - pali-language-research
+year: 1990
+publisher: pts
+address: "Oxford"
+pages: 36
+---
+
+A collection of K.R. Norman's five-part series of articles on the different meters found in the Lakkhaṇasutta of the Dīgha-nikāya.
```
Comment (buddhist-uni) [inline _content/articles/metres-of-the-lakkhana-suttanta_norman.md:17]: Link to DN30 (and possibly vise versa)

**Review by buddhist-uni — COMMENTED** [review]
Let's go with "booklets" for this

**Comment by JJM317** [comment]
Ok, I tried linking to DN 30 too. Success?

**Pull request merged**